app/inputs/nested_select_input.rb

```diff
@@ -94,11 +94,11 @@ def select_classes(level_data)
   end
 
   def model_name
-    @object.class.to_s.downcase
+    @object.class.to_s.underscore.gsub('/', '_')
```

Comment:
Yes I have for ex. Promotions::Owner and it crash javascript when reproduced in promotions::owner